### PR TITLE
📙 added fastq split instruction for COV-IRT

### PR DESCRIPTION
**Pull request by migbro**
:hammer: added ability to use gtf instead of cache for VEP
:book: updated PR template

# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

VEP update run successfully on Cavatica

**Test Configuration**:
* Environment:
* Test files:

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: fq_split_rg_instructions.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    --readFilesCommand zcat \
+    --runThreadN 24 \
+    --chimOutType Junctions \
+    --chimOutJunctionFormat 1
+    --chimSegmentMin 20 \
+    --outSAMtype BAM SortedByCoordinate \
+    --quantMode TranscriptomeSAM GeneCounts \
+    --outSAMheaderHD @HD VN:1.4 SO:coordinate \
+    --outFileNamePrefix /path/to/output/directory/${sample} \
+    --readFilesIn COVHA-P1-D06-N.filtered.human_H5NKGDSXY_1_R1.fq.gz,COVHA-P1-D06-N.filtered.human_H5NKGDSXY_2_R1.fq.gz,COVHA-P1-D06-N.filtered.human_H5NKGDSXY_1_R2.fq.gz,COVHA-P1-D06-N.filtered.human_H5NKGDSXY_2_R2.fq.gz
```
Comment (kmhernan) [inline fq_split_rg_instructions.md]: this isn't right, theere is a space between R1 and R2 lists

File: fq_split_rg_instructions.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+`pip3 install virtualenv # depends on your set up`
+
+```sh
+virtualenv venv --python /usr/bin/python3 # again, wherever your python3 (should be version 3.5+) is installed
+./venv/bin/pip install .
+venv/bin/gdc-fastq-splitter -h # if successful, you should get help menu
+```
+
+# Fun time run time
+
```
Comment (kmhernan) [inline fq_split_rg_instructions.md:19]: should note that when there are paired-end reads (2 fastqs) it will use 2 cores

**Review by kmhernan — CHANGES_REQUESTED** [review]
couple changes

File: fq_split_rg_instructions.md (hunk 1 above)
Comment (migbro) [reply fq_split_rg_instructions.md]: Ah yes. Reading is essential. Separated with read 1 and read 2 fastq lists with s space as STAR manual dictates. Thanks!

File: fq_split_rg_instructions.md (hunk 2 above)
Comment (migbro) [reply fq_split_rg_instructions.md:19]: Added comment

**Review by kmhernan — APPROVED** [review]

**Pull request merged**